[ch80] Update readme (#10)
Cobda/cobda-web: README.md

> **Cobda** is a streetwear e-commerce website where sellers and buyers meet each other with ease, driven by our desire to make community-like websites available to everyone in a friendly way. **Make certain** with a powerful AI function that automatically takes out inappropriate and irrelevant content to make sure your vibe goes as smoothly as possible. **Buy with confidence** by looking at product reviews and buyer's profile. This website runs perfectly on any online device.

Reviewer: ```suggestion
**Cobda** is a streetwear e-commerce website where sellers and buyers met with ease. With a community alike design, it's available for every genders and generations. **Make certain** with an autonomous feature that validate newly registered products at instant. Selling product is now faster than ever. **Buy with confidence** guaranteed by other customers. The review system builds up credits for the seller while giving confidence to the buyer vice versa. Finally, accessible through any devices with an internet connection 😎
```

Author: ```suggestion
**Cobda** is a streetwear e-commerce website where sellers and buyers met with ease. With a community alike design, it's available for every gender and generation. **Make certain** with an autonomous feature that validates newly registered products instantly. Selling product is now faster than ever. **Buy with confidence** guaranteed by other customers. The review system builds up credits for the seller while giving confidence to the buyer vice versa. Moreover, accessible through any devices with an internet connection.
```

> **Cobda** is a community-like e-commerce website for selling and buying streetwears. Plus, it's accessible through any online devices.

Reviewer 2: Grammar check 👍🏻
```suggestion
**Cobda** is a streetwear e-commerce website where sellers and buyers meet each other with ease, driven by our desire to make community-like websites available to everyone in a friendly way. **Make certain** with a powerful AI function that automatically takes out inappropriate and irrelevant content to make sure your vibe goes as smoothly as possible. **Buy with confidence** by looking at product reviews and buyer's profile. This website runs perfectly on any online device. 
```

> **Cobda** is a community-like e-commerce website for selling and buying streetwear stuff. Plus, it's accessible anywhere and anytime through the internet.

Reviewer: ```suggestion
**Cobda** is a community-like e-commerce website for selling and buying streetwears. Plus, it's accessible through any online devices.
```

> ## Prerequisite

Reviewer: ```suggestion
## Prerequisites
```

> You must download the following to be able to run.

Reviewer: I don't think this sentence is necessary this the title already has its own meaning.

- `Node.js (6.x or higher)` [download site](https://nodejs.org/en/download/package-manager/)

> - `local environment variable` **Please ask project owner privately for the environment variables.**

Reviewer: ```suggestion
- `.env` **Please ask project owner privately for the environment variables.**
```
